staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go

```diff
@@ -1378,7 +1378,11 @@ func (ss *scaleSet) GetNodeNameByIPConfigurationID(ipConfigurationID string) (st
 	matches := vmssIPConfigurationRE.FindStringSubmatch(ipConfigurationID)
 	if len(matches) != 4 {
 		klog.V(4).Infof("Can not extract scale set name from ipConfigurationID (%s), assuming it is managed by availability set", ipConfigurationID)
-		return "", "", ErrorNotVmssInstance
+		name, rg, err := ss.availabilitySet.GetNodeNameByIPConfigurationID(ipConfigurationID)
```

Comment:
could we log the returned err here?

Reply:
sure, will log and return the exact error

Reply:
I check if the vmas is disabled before calling the vmas method. If it is good I will do the same thing in the out-of-tree repo.

Reply:
LGTM